app/assets/javascripts/analytics.js.erb

```diff
@@ -27,6 +49,7 @@ var analyticsInit = function() {
       ga('govuk_shared.set', 'anonymizeIp', true);
       ga('govuk_shared.set', 'allowAdFeatures', false);
       ga('govuk_shared.linker:autoLink', ['www.gov.uk']);
+      ga('govuk_shared.set', 'location', stripPII(window.location.href));
```

Comment:
Is there a way to remove all query parameters from the URLs we send GA? Just in case these get renamed in future or we add others that are also PII but forget to strip them here.

Reply:
@bilbof I did think this, but the technical guidance directed to replace the params with strings so I followed that. I will check with a performance analyst to see if there is any reason for preserving the query strings, otherwise I can remove them.

Reply:
@bilbof for Harriet:

"So it's actually better to have [redacted-number] because it shows that there is missing data and what the redacted data is". I'll look into ways of testing it now.

Reply:
Thanks @Rosa-Fox!

Reply:
@bilbof I looked into testing with Jest... it seems possible but we will need to update our pipeline to run the JS tests. I will write a separate card so that we can get this part merged quicker.

Reply:
This is how you might get the URL without query params: https://stackoverflow.com/a/6257480